groclient/client.py

```diff
@@ -1604,6 +1604,43 @@ def search_for_entity(self, entity_type, keywords):
             )
             return result["id"]
 
+    def weight_area(self, series_name, weights):
```

Comment:
i think sticking to the "get_x" naming convention would make it more obvious to what the function actually does and fit with the existing functions like get_data_points, get_data_series, etc. "get_area_weighted_series"?

Reply:
Agree, updated